tests/utils/test_emails.py

```diff
@@ -0,0 +1,29 @@
+from django.core.mail.message import EmailMessage
+from factory import Faker
+
+from itou.utils.tasks import AsyncEmailBackend
+
+
+class TestAsyncEmailBackend:
+    def test_creates_one_task_per_message(self, mailoutbox):
```

Comment:
Le nom du test est peut-être un peu fallacieux car on ne vérifie jamais le nombre de tâche juste le nombre d'emails générés, mais je ne suis pas sûr qu'on puisse en _immediate mode_ ou alors peux-être avec un `mocker.spy()` :shrug:.
```suggestion
    def test_sanitize_mailjet_recipients_is_used(self, mailoutbox):
```

Reply:
J’ai remplacé par un commentaire dans le backend, et renommé le test.